vcbuild.bat

```diff
@@ -337,7 +337,8 @@ if errorlevel 1 (
 if "%target%" == "Clean" goto exit
 
 :after-build
-rd %config%
+:: Suppress "The system cannot find the file specified." on every clean build.
+rd %config% > nul 2> nul
 if errorlevel 1 echo "Old build output exists at 'out\%config%'. Please remove." & exit /B
```

Comment:
`rd %config% -ErrorAction SilentlyContinue` I think is a better option instead of redirecting the output.

---

Also, I wonder if the next statement works since the code is:

- trying to remove the code
- if can't, it should exit the dir.

If your build was successful, so there's something wrong with that `if` statement, or I'm wrong?

Maybe:

```powershell
if exists %config% echo "Old build output exists at 'out\%config%'. Please remove." & exit /B
```

Reply:
Maybe you mixed the environment of script execution. `vcbuild.bat` is executed in cmd not powershell.

![image](https://github.com/nodejs/node/assets/2276718/bc35cd99-45b2-49bc-a953-6cf9ab3cd985)

And for a clean build, it's known %config% is not exists. And for the incremental build, the %config% is just a symbol link (Directory Junction). It should be safe to remove it.

Reply:
You are right, I forgot about this is running on cmd.